config/prow/cluster/build/create-loop-devs_daemonset.yaml

```diff
@@ -40,7 +40,7 @@ spec:
             done
             sleep 100000000
           done
-        image: alpine:3.6
+        image: gcr.io/k8s-prow/alpine:latest
```

Comment:
@ameukam @dims I don't think is a good idea to depend on `latest`tag, as it is going to be a moved target and it will be always pulled despite it is in the node, don't we have another image of alpine with a version we can pin?

Reply:
we have control on this image unlike the public one. So any change landing of the prow-owned alpine is approved by EngProd or the (testing|k8s-infra) leads. 
we can also pin this image:

```bash
crane ls gcr.io/k8s-prow/alpine
0.1
latest
v20200604-5b1db6c408-dirty
v20200605-44f6c96
v20200713-e9b3d9d
v20200824-5d057db
v20210618-2814345
v20210818-e6fa606
v20211015-92dc282
v20220314-46af1b01a6
v20220505-bca727ee56
v20230718-8d3170488d
```

Happy to reverse this commit if SIG testing make the call.

Reply:
no need to reverse, just pin it lgtm

Reply:
See https://github.com/kubernetes/test-infra/pull/31653